src/main/kotlin/com/dansplugins/factionsystem/listener/BlockBreakListener.kt

```diff
@@ -47,6 +47,18 @@ class BlockBreakListener(private val plugin: MedievalFactions) : Listener {
                 event.player.sendMessage("$RED${plugin.language["CannotBreakBlockInFactionTerritory", claimFaction.name]}")
             }
         }
+
+        // if block locked and breaker is owner, unlock
+        val lockService = plugin.services.lockService
+        val lockedBlock = lockService.getLockedBlock(blockPosition) ?: return
+        if (lockedBlock.playerId == mfPlayer.id) {
+            lockService.unlock(event.player, event.block)
+        }
+        else {
```

Comment:
else should be on the same line as the closing brace

Reply:
Fixed!